### PR TITLE
ongoing updates to CFT spec

### PR DESCRIPTION
**Pull request by ledhed2222**
A variety of updates to the spec. Each update is clearly marked by a different commit.

### CODE REVIEW

**Comment by shawnxie999** [comment]
should clawback flag also be added as part of the spec?

**Review by intelliot — APPROVED** [review]

**Review by sappenin — CHANGES_REQUESTED** [review]

**Review by intelliot — CHANGES_REQUESTED** [review]
See requests by @sappenin

File: XLS-0033d-CFTs/README.md [hunk 1]
```diff
@@ -102,15 +101,15 @@ A set of flags indicating properties or other options associated with this **`CF
...
+| `lsfCFTRequireAuth` | ️`0x0004`  | If set, indicates that _individual_ holders must be authorized. This enables issuers to limit who can hold their assets.  |
+| `lsfCFTCanEscrow`             | `0x0008`  | If set, indicates that _individual_ holders can place their balances into an escrow. |
+| `lsfCFTCanTrade`              | `0x0010`  | If set, indicates that _individual_ holders can trade their balances using the XRP Ledger DEX or AMM.
+| `lsfCFTCanTransfer`          | ️`0x0020`  | If set, indicates that tokens held by non-issuers may be transferred to other accounts. If not set, indicates that tokens held by non-issuers may not be transferred except back to the issuer; this enables use-cases like store credit. |
+| `lsfCFTCanClawback`         | ️`0x0040`  | If set, indicates that the issuer may use the `Clawback` transaction to clawback value from _individual_ holders.|
```
Comment (shawnxie999) [inline XLS-0033d-CFTs/README.md:110]: i may have missed it. Why are we adding the `CFT` prefix to the flag names? Wouldn't it make the flags values cleaner and also usable in the future for other objects by not putting the prefix?

File: XLS-0033d-CFTs/README.md [hunk 2]
```diff
@@ -102,15 +101,15 @@ A set of flags indicating properties or other options associated with this **`CF
...
 
-With the exception of the `lsfFrozen` flag, which can be mutated via the `**CFTokenIssuanceSet**` transactions, these flags are **immutable**: they can only be set during the **`CFTokenIssuanceCreate`** transaction and cannot be changed later.
+With the exception of the `lsfLocked` flag, which can be mutated via the `**CFTokenIssuanceSet**` transaction, these flags are **immutable**: they can only be set during the **`CFTokenIssuanceCreate`** transaction and cannot be changed later.
```
Comment (shawnxie999) [inline XLS-0033d-CFTs/README.md:112]: should be `lsfCFTLocked` if this is the new name we are going forward with

File: XLS-0033d-CFTs/README.md (hunk 1 above)
Comment (ledhed2222) [reply XLS-0033d-CFTs/README.md:110]: there was a comment on my creation implementation PR against re-using flag IDs, which I support. Therefore I think it is important to have flags that denote their specifity.

**Pull request closed without merging**